src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -207,6 +207,7 @@ public function getConfigTreeBuilder()
                             ->info('Enable the tags-based cache invalidation system.')
                             ->canBeEnabled()
                             ->children()
+                                ->scalarNode('purger')->defaultValue('varnish')->cannotBeEmpty()->info('The name of the purger to use.')->end()
```

Comment:
IMO, it should be directly the name of the service instead of a tricky concatenation (i.e `api_platform.http_cache.purger.$purger`). Or did I miss something specific?
So, create an implementation of your new interface for Varnish whose the service name will be the default value here. 

WDYT?

Reply:
Addressed in 2e5bbbf4c16a6a42caa25f53fb3abf96a1d62573.